.eslintrc

```diff
@@ -4,7 +4,8 @@
     "sourceType": "module"
   },
   "extends": [
-    "eslint:recommended"
+    "eslint:recommended",
+    "prettier"
```

Comment:
As per https://github.com/prettier/eslint-config-prettier?tab=readme-ov-file#installation

Reply:
These rules work well with prettier + I assume they were added for a reason, so why not keep them, that's my thinking. I could get rid of all of the rules here if  you want so, but the main thing I wanted to do with this change was to unify the style.